community_supplied/Protocols/Bgp/check-bgp-summary-netconf.rule

```diff
@@ -0,0 +1,174 @@
+                }
+            }
+            /*
+            * Fields defined using formulae by user-defined function
+            */
+            field neighbor-check {
+                formula {
+                    user-defined-function {
+                        function-name neighbor-check;
+                        argument neighbor_list "@neighbor";
+                        argument peer_state "$peer-address";
+                    }
+                }
+                type integer;
+            }
+            /*
+            * Field defined using sensor path. Map the longer sensor names
+            * to the shorter field names used in the rules.
+            */
+            field peer-address {
+                sensor bgp-sensor {
+                    where "peer-address =~ /^{{peer-address-var}}$/";
+                    path peer-address;
+                }
+                type string;
+                description "Neighbor address";
+            }
+            field peer-state {
+                sensor bgp-sensor {
+                    path peer-state;
+                }
+                type string;
+                description "Neighbor state";
+            }
+            /*
+            * Anomaly detection logic.
+            */
+            trigger bgp-neighbor {
+                synopsis "BGP neighbors";
+                description "Sets health based on the change in BGP neighbor and neighbor state";
+                frequency 90s;
+                /*
+                * Sets color to green and sends out an anomaly notification when
+                * neighbor check ($neighbor-check) is equal to 1 and neighbor
+                * state is equal to Established.
+                */
+                term neighbor-is-configured {
+                    when {
+                        equal-to "$neighbor-check" 1;
+                        matches-with "$peer-state" Established;
+                    }
+                    then {
+                        status {
+                            color green;
+                            message "Neighbor address ($peer-address) is configured. Neighbor state is Established.";
+                        }
+                    }
+                }
+                /*
+                * Sets color to yellow and sends out an anomaly notification when
+                * neighbor check ($neighbor-check) is equal to 1 and neighbor
+                * state is not equal to Established.
+                */
+                term neighbor-is-configured-and-state-is-not-equal-to-established {
+                    when {
+                        equal-to "$neighbor-check" 1;
+                        does-not-match-with "$peer-state" Established;
+                    }
+                    then {
+                        status {
+                            color yellow;
+                            message "Neighbor address ($peer-address) is configured. Neighbor state ($peer-state) is not Established.";
```

Comment:
change message to"Neighbor address ($peer-address) is configured. Neighbor state is $peer-state."